controllers/api/userRoutes.js

```diff
@@ -21,7 +22,7 @@ router.post("/", async (req, res) => {
       }
 });
 
-router.post('/login', async (req, res) => {
+router.post('/login', withAuth, async (req, res) => {
```

Comment:
I don't think we want withAuth here, do we? This is the route to log the user in; if they're hitting this route, we already know they're not logged in.

Reply:
Agreed. Just removed that!